common/lib/capa/capa/responsetypes.py

```diff
@@ -248,7 +248,9 @@ def render_html(self, renderer, response_msg=''):
           - response_msg: a message displayed at the end of the Response
         """
         # render ourself as a <span> + our content
-        tree = etree.Element('span')
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
```

Comment:
@ehteshamkafeel The author "should" customize the Display name value for each component on the page (but they usually don't).  If they didn't customize it, and there ended up being a bunch of non-unique section titles, it wouldn't be terrible, and more importantly, the accessibility issue would not be the result of our platform, but an authoring error (which we can write documentation to help them avoid doing.)  

If you think you can reliably count the number of these components on a page, then lets go with that.  If not, lets go with Display name, and we can help the author try to avoid issues in documentation.

I also want to verify what each section element contains here.  Is it "Problems" or "Questions"?  If its problems, which can have multiple questions, lets use Problem 1, Problem 2, etc.  instead of Question 1...  Sorry for the confusion. I often use them interchangeably.